script-base.js

```diff
@@ -23,19 +31,14 @@ var Generator = module.exports = function Generator() {
     this.env.options.appPath = this.options.appPath;
 
     if (!this.env.options.appPath) {
-      try {
-        this.env.options.appPath = require(path.join(process.cwd(), 'bower.json')).appPath;
-      } catch (e) {}
+      this.env.options.appPath = bowerJson.appPath;
     }
     this.env.options.appPath = this.env.options.appPath || 'app';
```

Comment:
remove 33-35 and change 36 to:

`this.env.options.appPath = this.env.options.appPath || bowerJson.appPath || 'app';`